packages/editor/src/client/blocks/htmlBlock.tsx

```diff
@@ -19,6 +20,10 @@ export const HTMLBlock = ({value, onChange, autofocus}: BlockProps<HTMLBlockValu
     }
   }, [])
 
+  useEffect(() => {
+    executeScriptElements(document.getElementById('html-block') as HTMLElement)
```

Comment:
Problem with ID is that it's unique on the page, we should probably not use id here.

instead of document.getElementById we probably should use use `<ref>.querySelectorAll('.html-block')`  instead of document.